apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/TransactionPool.cs

```diff
@@ -115,6 +115,16 @@ public static TransactionOptions GetLastUsedTransactionOptions(this Session sess
             return null;
         }
 
+        public static bool IsPreWarmedTransactionReady(this Session session)
+        {
+            SessionInfo info;
+            if (s_sessionInfoTable.TryGetValue(session, out info))
+            {
+                return info.HasActiveTransaction;
+            }
+            return false;
+        }
+
```

Comment:
With C# 7 this can now be an expression-bodied method:
```csharp
... => s_sessionInfoTable.TryGetValue(session, out var info) && info.HasActiveTransaction;
```
But I don't feel too strongly about it.